pkg/kubectl/cmd/edit.go

```diff
@@ -354,7 +350,9 @@ func getPrinter(cmd *cobra.Command) (*editPrinterOptions, error) {
 	}
 }
 
-func getMapperAndResult(f cmdutil.Factory, args []string, options *resource.FilenameOptions, editMode EditMode) (meta.RESTMapper, *resource.Mapper, *resource.Result, string, error) {
+type resultGetter func([]byte) *resource.Result
```

Comment:
Interesting. I wasn't sure at the time if we'd need the direct result. I could switch it to that